README.md

```diff
@@ -58,19 +58,91 @@ Run WDL conformance tests.
 options:
   -h, --help            show this help message and exit
   --verbose             Print more information about a test
-  --versions VERSIONS, -v VERSIONS
+  --versions {1.0,1.1,draft-2}, -v {1.0,1.1,draft-2}
                         Select the WDL versions you wish to test against.
   --tags TAGS, -t TAGS  Select the tags to run specific tests
   --numbers NUMBERS, -n NUMBERS
-                        Select the WDL test numbers you wish to run.
+                        Select the WDL test numbers you wish to run. Can be a comma separated list or hyphen separated inclusive ranges. Ex: -n=1-4,6,8-10
   --runner RUNNER, -r RUNNER
                         Select the WDL runner to use.
-  --threads THREADS     Number of tests to run in parallel.
+  --threads THREADS     Number of tests to run in parallel. The maximum should be the number of CPU cores (not threads due to wall clock timing).
+  --time                Time the conformance test run.
+  --quiet
+  --exclude-numbers EXCLUDE_NUMBERS
+                        Exclude certain test numbers.
+  --toil-args TOIL_ARGS
+                        Arguments to pass into toil-wdl-runner. Ex: --toil-args="caching=False"
+  --miniwdl-args MINIWDL_ARGS
+                        Arguments to pass into miniwdl. Ex: --miniwdl-args="--no-outside-imports"
+  --cromwell-args CROMWELL_ARGS
+                        Arguments to pass into cromwell. Ex: --cromwell-args="--options=[OPTIONS]"
+  --id ID               Specify WDL tests by ID.
+  --repeat REPEAT       Specify how many times to run each test.
+  --jobstore-path JOBSTORE_PATH, -j JOBSTORE_PATH
+                        Specify the PARENT directory for the jobstores to be created in.
+  --progress            Print the progress of the test suite as it runs.
+
 ```
+Invoking `run.py` with no options will result in the entire test suite being ran, which can take a long time.
+Including `--progress` is recommended when running over a long period of time.
+
+Certain tests can be specified with `--id`, `--tags`, and `--numbers`:
+```commandline
+python run.py --number 1 --id md5 --tags stderr --version 1.0 --runner toil-wdl-runner
+Testing runner toil-wdl-runner on WDL versions: 1.0
+
 
+=== REPORT ===
+
+1: SUCCEEDED: Standard Lib: Basic test for stderr()
+Iteration: 1
+40: SUCCEEDED: Legacy test for stderr_as_output
+Iteration: 1
+68: SUCCEEDED: MD5 test
+Iteration: 1
+3 tests run, 3 succeeded, 0 failed, 0 skipped
+```
+`--repeat` specifies how many times to run each test. `--threads` allows multiple tests to run simultaneously;
+this should be set to the number of processors and not threads in a computer if running with `--time` or with the performance testing script.
```

Comment:
There's no machinery currently to reserve certain threads. Most if not all of the tests only allocate 1 cpu core, but I don't think the behavior of how to run the workflow/task section on what cores is strictly defined (as in, if task allocates 1 cpu, it might run on a separate thread than the workflow). I think the best way is to tweak the documentation to imply that the maximum amount of tests to run is the number of threads excluding hyperthreading, and that consistent timings may not be fully guaranteed if a runner uses more threads as specified and once all threads are saturated.